en/99-apendice/2-repository-rules.md

```diff
@@ -0,0 +1,96 @@
+## Repository rules
+El principal material con el que trabajamos es código, con lo que se vuelve menester para la salud de todos los proyectos (y la nuestra) que mantengamos una serie de reglas para trabajar ordenadamente en los repositorios.
+
+### Gitflow
+Seguimos [gitflow](http://nvie.com/posts/a-successful-git-branching-model/) como modelo de branching y tagging, donde `master` actúa como branch de develop y mantenemos un branch `stable`.
```

Comment:
We follow [gitflow](http://nvie.com/posts/a-successful-git-branching-model/) for a guideline to branching and tagging. `master` acts as the development branch and there's an additional `stable` one.